examples/src/legacy/basic_encryption.py

```diff
@@ -1,29 +1,19 @@
-# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"). You
-# may not use this file except in compliance with the License. A copy of
-# the License is located at
-#
-# http://aws.amazon.com/apache2.0/
-#
-# or in the "license" file accompanying this file. This file is
-# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
-# ANY KIND, either express or implied. See the License for the specific
-# language governing permissions and limitations under the License.
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
 """Example showing basic encryption and decryption of a value already in memory."""
 import aws_encryption_sdk
 
 
-def cycle_string(key_arn, source_plaintext, botocore_session=None):
+def run(aws_kms_cmk, source_plaintext, botocore_session=None):
```

Comment:
Everyone else probably knows, but why set botocore_session to none here?

Reply:
I'm going to be updating the legacy examples to match the format (comments/layout/etc) of the new examples once the new ones are ready. As part of that I'm actually going to be pulling that as input. That was a case of me trying to squeeze too many examples of possible configuration changes into a single example. The better place for that will be an example that specifically focuses on how to configure the KMS MKP/keyring.